selfdrive/car/ford/interface.py

```diff
@@ -14,7 +14,7 @@ class CarInterface(CarInterfaceBase):
   @staticmethod
   def _get_params(ret, candidate, fingerprint, car_fw, experimental_long, docs):
     ret.carName = "ford"
-    ret.dashcamOnly = candidate in {CAR.F_150_MK14}
+    ret.dashcamOnly = candidate in {CAR.F_150_MK14, CAR.F_150_LIGHTNING_MK1, CAR.MUSTANG_MACH_E_MK1}
```

Comment:
```suggestion
    ret.dashcamOnly = candidate in CANFD_CAR
```
?

Reply:
Handled in https://github.com/coffee-cake-studios/openpilot/commit/bb34a29347b5e0f9bce4e2dd4d42a3df43cd60e3